cassandra-diagnostics-core/src/main/java/io/smartcat/cassandra/diagnostics/module/slowquery/SlowQueryLogDecider.java

```diff
@@ -50,8 +50,7 @@ public boolean isForReporting(Query query) {
     private boolean executionTimeForLogging(long executionTimeInMilliseconds) {
         logger.trace("Checking if execution time:{} is above threshold: {}", executionTimeInMilliseconds,
                 slowQueryConfiguration.slowQueryThreshold());
-        if (slowQueryConfiguration.logAllQueries()
-                || executionTimeInMilliseconds >= slowQueryConfiguration.slowQueryThreshold()) {
+        if (executionTimeInMilliseconds > slowQueryConfiguration.slowQueryThreshold()) {
```

Comment:
Case of logging everything greater than zero is valid because it basically logs all queries that took longer than 1ms to execute.